src/components/GearOptimizer.jsx

```diff
@@ -98,6 +99,7 @@ const MainComponent = ({ data }) => {
           <Controls profession={profession} />
 
           <ResultTable />
+          <SharingSection />
```

Comment:
This should only be displayed if a character is selected. Is it a performance killer to use `useSelector` on character here?

Reply:
Hmm. I think to compartmentalize it, having `SharingSection` just return nothing if there's no selected character makes the most sense!